src/pages/_home.mdx

```diff
@@ -12,5 +12,5 @@ The main objective of the DAS lab is to investigate, develop and deploy data-dri
 We also apply state-of-the-art Data Mining, Artificial Intelligence, and Statistical Analysis techniques in order to build our solutions. Several of the techniques and tools developed by DAS members have been developed in cooperation with and/or being used in some of the world’s largest software companies.
 
 :::info
-**The DAS is recruiting!** The DAS lab has several open positions at the [Master's](https://dearprof.com/positions/663a917eb1a073f28c471c45/apply) and [PhD](https://dearprof.com/positions/663a90b6b1a073f28c471c42/apply) level.
+**The DAS is recruiting!** Apply for open positions here: [Master's](https://dearprof.com/positions/663a917eb1a073f28c471c45/apply) and [PhD](https://dearprof.com/positions/663a90b6b1a073f28c471c42/apply).
```

Comment:
What about "The DAS Lab is recruiting" ....

Reply:
The text is from Emad as is.